playbooks/common/openshift-master/config.yml

```diff
@@ -334,8 +334,8 @@
   serial: 1
   vars:
     sync_tmpdir: "{{ hostvars.localhost.g_master_mktemp.stdout }}"
-    openshift_master_ha: "{{ groups.oo_masters_to_config | length > 1 }}"
-    openshift_master_count: "{{ groups.oo_masters_to_config | length }}"
+    openshift_master_ha: "{{ groups.oo_masters | length > 1 }}"
```

Comment:
Either that or have openshift_facts throw an error if the value is different than the persisted setting?